Dockerfile

```diff
@@ -47,4 +47,4 @@ COPY ./ .
 RUN rm -Rf /model-res-avm/renv && \
     mv /setup/renv /model-res-avm/renv
 
-CMD dvc pull && dvc repro
+CMD ["sh", "-c", "dvc pull && dvc repro"]
```

Comment:
**[Thought, non-blocking]** I have a vague memory of struggling with the array version of this command, possibly when configuring Batch to override it. No action to take right now since I can't remember the problem clearly, but just a heads up for things to watch out for down the line.